exporters/elasticsearch-exporter/src/main/resources/zeebe-record-decision-requirements-template.json

```diff
@@ -0,0 +1,54 @@
+{
+  "index_patterns": [
+    "zeebe-record_decision-requirements_*"
+  ],
+  "composed_of": ["zeebe-record"],
+  "priority": 20,
+  "version": 1,
+  "template": {
+    "settings": {
+      "number_of_shards": 1,
+      "number_of_replicas": 0,
+      "index.queries.cache.enabled": false
+    },
+    "aliases": {
+      "zeebe-record-decision-requirements": {}
+    },
+    "mappings": {
+      "properties": {
+        "value": {
+          "dynamic": "strict",
+          "properties": {
+            "decisionRequirementsId": {
+              "type": "keyword"
+            },
+            "decisionRequirementsName": {
+              "type": "keyword"
+            },
+            "decisionRequirementsVersion": {
+              "type": "long"
+            },
+            "decisionRequirementsKey": {
+              "type": "long"
+            },
+            "namespace": {
+              "type": "keyword"
+            },
+            "resourceName": {
+              "type": "text"
+            },
```

Comment:
I'm not all that familiar with ES type families, but after reading about it a bit, I understand that the difference between `text` and `keyword` is that keywords are not analyzed and thus stay as they are while text is analyzed. So `text` gets tokenized using an analyzer, which by default lowercases and splits the string into words. 

❓ What that exactly means is still unclear to me, but it makes me wonder why `resourceName` would have to be a `text` while the other strings `decisionRequirementsName`, `namespace`, etc. are `keywords`. Can you clarify this for me?

Reply:
> Can you clarify this for me?

Reply:
💭 Not the best idea that we maintain this exporter 😄